src/ethereum/frontier/spec.py

```diff
@@ -860,12 +860,10 @@ def calculate_block_difficulty(
     Computes difficulty of a block using its header and
     parent header.
 
-    Since the genesis block is the first on the chain and has no parent,
-    its difficulty is a predefined value stored as ``GENESIS_DIFFICULTY``.
-    For all other blocks the difficulty is determined by the time the
-    block was created after its parent. If a block's timestamp is more
-    than 13 seconds after its parent block then its difficulty is set
-    as the difference between the parent's difficulty and the
+    The difficulty of a block is determined by the time the block was
```

Comment:
I think you should still note that the genesis block is a special case somewhere, and that its difficulty is `MINIMUM_DIFFICULTY`.

Reply:
The difficulty of the genesis block is bigger than `MINIMUM_DIFFICULTY`. The fact that the constant was previously called `GENESIS_DIFFICULTY` was an outright bug.